### PR TITLE
templates: add 'Environment' as first section

### PR DESCRIPTION
**Pull request by taroth21**
- for now, cover the products and versions in the text
- entitled the section 'Environment' to keep similar to
  knowledgebase articles
- added ID proposal for section

### CODE REVIEW

File: templates/xml/concept.xml [hunk 1]
```diff
@@ -51,6 +51,31 @@
    </dm:docmanager>
  </info>
 
+ <section xml:id="environment-">
```
Comment (tomschr) [inline templates/xml/concept.xml:54]: Should we add a comment or placeholder here? I guess the writer has to amend that, right?

File: templates/xml/concept.xml [hunk 2]
```diff
@@ -51,6 +51,31 @@
...
+  <title>Environment</title>
+  <para>This document applies to the following products and product versions:</para>
+  <itemizedlist>
+   <listitem>
+    <para>&sles;&nbsp;15&nbsp;SP3/15&nbsp;SP2/15&nbsp;SP1/15&nbsp;GA/12&nbsp;SP5/12&nbsp;SP4/12&nbsp;SP3</para>
```
Comment (tomschr) [inline templates/xml/concept.xml:59]: Maybe we can find something which is a bit more readable? I fear with the slashes, it _can_ give an overly long line which is hard to break. Especially if you use NBSPs. What do using commas and spaces like this:

```
SLES 15  SP3, 15 SP2, ...
```

**Review by tomschr — COMMENTED** [review]
I think it looks fine. Some minor ideas and comments.

For the time being this is probably enough. When I wrote that, I had a more machine-readable idea in mind. This can make it easier to be used in search engines or added as layout information. Be it concise as a short label or as a whole section.

TL;DR: It's okay. But we should revisit this issue later and find a way to use some metadata and maybe generate some text automatically.

File: templates/xml/concept.xml (hunk 2 above)
Comment (taroth21) [reply templates/xml/concept.xml:59]: Good argument with the line length, I'll change it to commas (I assume I can keep the NBSPs then because the line break could be done after each comma at least). Or would you remove them, Toms?

File: templates/xml/concept.xml (hunk 1 above)
Comment (taroth21) [reply templates/xml/concept.xml:54]: I'll add an XML comment, thanks!

File: templates/xml/concept.xml (hunk 2 above)
Comment (taroth21) [reply templates/xml/concept.xml]: Clarified via chat: keep NBPs.
